### PR TITLE
add Spark 3.0 changes

### PR DESCRIPTION
**Pull request by lanking520**
Add Spark 3.0 and image loading capability

### CODE REVIEW

File: spark/3.0/image-classification/README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# DJL Spark Image Classification Example
```
Comment (frankfliu) [inline spark/3.0/image-classification/README.md:1]: change 3.0 to spark3.0 to make it clear

File: spark/3.0/image-classification/README.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+```
+runtimeOnly "ai.djl.tensorflow:tensorflow-model-zoo:0.8.0"
+runtimeOnly "ai.djl.tensorflow:tensorflow-native-auto:2.3.0"
+```
+
+Apart from that, you may also need to use `NaiveEngine` mode in MXNet for multi-thread inference.
+Please add this line in your `SparkConf`
+
+```
+.setExecutorEnv("MXNET_ENGINE_TYPE", "NaiveEngine")
```
Comment (frankfliu) [inline spark/3.0/image-classification/README.md:49]: Should we mention: OMP_NUM_THRRAD?

File: spark/3.0/image-classification/build.gradle [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+group "com.example"
+version "1.0-SNAPSHOT"
+
+repositories {
+    jcenter()
+}
+
+sourceCompatibility = 1.8
```
Comment (frankfliu) [inline spark/3.0/image-classification/build.gradle:13]: remove these two line, doesn't belong here

**Review by frankfliu — APPROVED** [review]

File: spark/spark3.0/image-classification/build.sbt [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+scalaVersion := "2.12.10"
+scalacOptions += "-target:jvm-1.8"
+
+resolvers += Resolver.jcenterRepo
+
+libraryDependencies += "org.apache.spark" %% "spark-sql" % "3.0.1"
+libraryDependencies += "org.apache.spark" %% "spark-mllib" % "3.0.1"
+libraryDependencies += "ai.djl" % "api" % "0.8.0"
+
+// libraryDependencies += "ai.djl.mxnet" % "mxnet-model-zoo" % "0.8.0"
```
Comment (stu1130) [inline spark/spark3.0/image-classification/build.sbt:14]: add tensorflow dependencies here as comment

**Review by stu1130 — APPROVED** [review]
in general LGTM

**Pull request merged**